content/en/users/im/_index.md

```diff
@@ -3,7 +3,7 @@ title: "Infrastructure Manager"
 type: docs
 weight: 45
 description: >
-  How to use Infrastructure Manager (IM)
+  Using Infrastructure Manager on EGI Cloud
```

Comment:
A better subtitle is needed. Maybe "Automating deployment of virtual infrastructures in the EGI Cloud".

Reply:
changed.